packages/desktop-client/src/components/sidebar/ItemContent.tsx

```diff
@@ -21,7 +21,7 @@ function ItemContent({
   forceActive,
   children,
 }: ItemContentProps) {
-  return onClick ? (
+  return onClick || !to ? (
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I don't think we need this change.